CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 * [ENHANCEMENT] Querier: Implement result caching for tenant query federation. #3640
 * [BUGFIX] Allow `-querier.max-query-lookback` use `y|w|d` suffix like deprecated `-store.max-look-back-period`. #3598
 * [BUGFIX] Memberlist: Entry in the ring should now not appear again after using "Forget" feature (unless it's still heartbeating). #3603
+* [BUGFIX] Ruler: Rule group limit enforcement should now allow the same number of rules in a group as the limit. #3616
```

Comment:
We've just released `1.7.0-rc.0`. Could you rebase `master` and move the CHANGELOG entry under the `master / unreleased` section, please?

Reply:
Cortex release 1.8.0 is now in progress. Could you rebase master and move the CHANGELOG entry under the master / unreleased section, please?